src/lib.rs

```diff
@@ -145,6 +145,12 @@
 //! }
 //! ```
 
+#[cfg(all(PyPy, Py_LIMITED_API))]
+compile_error!(concat!(
+    "PyPy still does not support abi3. ",
+    "See https://foss.heptapod.net/pypy/pypy/-/issues/3397 for the discussion."
+));
```

Comment:
Just a couple of wording changes which I think makes it read easier:

```suggestion
compile_error!(
    "PyPy does not yet support abi3. \
    See https://foss.heptapod.net/pypy/pypy/-/issues/3397 for more information."
);
```